ng-dev/ts-circular-dependencies/analyzer.ts

```diff
@@ -32,13 +33,18 @@ const DEFAULT_EXTENSIONS = ['ts', 'js', 'd.ts'];
 export class Analyzer {
   private _sourceFileCache = new Map<string, ts.SourceFile>();
 
+  private _ignoreTypeOnlyChecks: boolean;
+
   unresolvedModules = new Set<string>();
   unresolvedFiles = new Map<string, string[]>();
 
   constructor(
     public resolveModuleFn?: ModuleResolver,
+    {ignoreTypeOnlyChecks}: CircularDependenciesParserOptions = {},
```

Comment:
Nit: Can we just pass through the flag explicitly? The analyzer should just take a boolean and we do the config handling in the CLI index, like with other options.

Reply:
I can do that, I switched them into two interfaces so that the interface loaded here is just the things the parser needs, I could generalize it to the analyzer potentially as well.


Thoughts?

Reply:
I would prefer just passing it through manually as a parameter, but this is just a nit anyway, and I'm good  as is.